src/test/java/com/michelin/ns4kafka/services/TopicServiceTest.java

```diff
@@ -984,6 +984,128 @@ void shouldTagsBeInvalidWhenNotConfluentCloud() {
         List<String> validationErrors = topicService.validateTags(ns, topic);
         assertEquals(1, validationErrors.size());
         assertEquals("Invalid value \"TAG_TEST\" for field \"tags\": tags are not currently supported.",
-            validationErrors.get(0));
+                validationErrors.getFirst());
+    }
+
+    @Test
+    void shouldReturnFalseWhenTagFormatIsCorrect() {
```

Comment:
As mentioned before, should return true if format is correct is preferable

Reply:
Right